tests/urdf_examples/piRobot.urdf

```diff
@@ -0,0 +1,512 @@
+<robot name="pi_robot">
```

Comment:
If you copied a urdf from somewhere you should properly attribute the owner and check that their licence permits this copy

Also what purpose does this urdf serve? It is not called from anywhere